src/main/resources/META-INF/patches/graphql/01-componentVisibility.started.graphql

```diff
@@ -5,6 +5,7 @@ mutation {
                 pcHide: value(name:"hideLegacyPageComposer", value:"true")
                 pbShow: value(name:"showPageBuilder", value:"true")
                 catManShow: value(name:"showCatMan", value:"true")
```

Comment:
"showCatMan" should be removed as it does not exist anymore

Reply:
Added comment: https://jira.jahia.org/browse/BACKLOG-21414?focusedCommentId=153646&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-153646